chat/src/main/java/com/github/twitch4j/chat/events/channel/DeleteMessageEvent.java

```diff
@@ -8,7 +8,7 @@
 import lombok.experimental.Accessors;
 
 /**
- * Fired when a single message was deleted over IRC by a moderator via /delete <target-msg-id>
+ * Fired when a single message was deleted over IRC by a moderator via /delete {@literal <}target-msg-id{@literal >}
```

Comment:
`&lt;` and `&gt;` is enough or
```suggestion
 * Fired when a single message was deleted over IRC by a moderator via {@code /delete &lt;target-msg-id&gt;}

Reply:
I took a look at the docs and if we use `@code`, then we don't need to use `&lt;` - so `{@code /delete <target-msg-id>}` would work too.
But the current version with literal is fine.